.github/workflows/deploy.yml

```diff
@@ -0,0 +1,72 @@
+name: Deploy NetKAN-Infra
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    if: ${{ github.repository == 'KSP-CKAN/NetKAN-Infra' }}
```

Comment:
Might be better to make this check whether the secrets are defined, like the one for CKAN does:

https://github.com/KSP-CKAN/CKAN/blob/e39d735583e2e965cc99e8196e8f76a99ea3c822/.github/workflows/deploy.yml#L71

```yml
if: ${{ env.DOCKERHUB_USERNAME && env.DOCKERHUB_PASSWORD }}
```

Reply:
I'm fine with being consistent across the projects.